.env.development

```diff
@@ -3,3 +3,4 @@ VUE_APP_TRACTION_GRAPHQL_URL=http://traction/graphql
 VUE_APP_PRINTMYBARCODE_BASE_URL=http://printmybarcode
 VUE_APP_SAMPLEEXTRACTION_BASE_URL=http://sampleextraction
 VUE_APP_LOG=true
+VUE_APP_TUBE_LABEL_TEMPLATE_NAME=traction_tube_label_template_v2
```

Comment:
Not related to this line but did we want the .local versions of the env files in the end? Checked into the repo I mean..